kbatch_papermill/__init__.py

```diff
@@ -1,8 +1,9 @@
 """
-Run notebooks via kbatch, store output in s3
+Run notebooks via kbatch, store output in S3.
 """
 
-from ._kbatch import *  # noqa
-from ._papermill import *  # noqa
+from ._kbatch import print_job_status, wait_for_jobs
+from ._papermill import kbatch_papermill
 
 __version__ = "0.1.0.dev"
+__all__ = ["print_job_status", "wait_for_jobs", "kbatch_papermill"]
```

Comment:
`__all__` should not be needed here when relying on `__all__` from the subpackages and `import *`

Reply:
Unfortunately, it looks like `autodoc` does not automatically fetch the submodules' definitions of `__all__`...
I will try to find whether someone has a work-around...

Reply:
I solved the issue by using the `::imported-members::` option, as suggested in this [issue](https://github.com/sphinx-doc/sphinx/issues/7433#issue-596033984); even though I am quite confused because to me, given its [definition](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-imported-members), it should be solved it...